website/docs/faq.md

```diff
@@ -3,4 +3,40 @@ title: FAQ
 ---
 
 ## What kind of vulnerabilities can Copa patch?
-Copa is capable of patching "OS level" vulnerabilities. This includes packages (like `openssl`) in the image that are managed by a package manager such as `apt` or `yum`. Copa is not currently capable of patching vulnerabilities at the "application level" such as Python packages or Go modules.
\ No newline at end of file
+Copa is capable of patching "OS level" vulnerabilities. This includes packages (like `openssl`) in the image that are managed by a package manager such as `apt` or `yum`. Copa is not currently capable of patching vulnerabilities at the "application level" such as Python packages or Go modules.
+
+## Can I replace the package repositories in the image with my own?
+
+:::caution
+
+Experimental: This feature might change without preserving backwards compatibility.
+
+:::
+
+Copa does not support replacing the repositories in the package managers with alternatives. Images must already use the intended package repositories. For example, for debian, updating `/etc/apt/sources.list` from `http://archive.ubuntu.com/ubuntu/` to a mirror, such as `https://mirrors.wikimedia.org/ubuntu/`.
+
+If you need the tooling image to use a different package repository, you can create a source policy to define a replacement image and/or pin to a digest. For example, the following source policy replaces `docker.io/library/debian:11-slim` image with `foo.io/bar/baz:latest@sha256:42d3e6bc186572245aded5a0be381012adba6d89355fa9486dd81b0c634695b5`:
+
+```shell
+cat <<EOF > source-policy.json
+{
+    "rules": [
+        {
+            "action": "CONVERT",
+            "selector": {
+                "identifier": "docker-image://docker.io/library/debian:11-slim"
+            },
+            "updates": {
+                "identifier": "docker-image://foo.io/bar/baz:latest@sha256:42d3e6bc186572245aded5a0be381012adba6d89355fa9486dd81b0c634695b5"
+            }
+        }
+    ]
+}
+EOF
+
+export COPA_SOURCE_POLICY=source-policy.json
```

Comment:
Also need to set `EXPERIMENTAL_BUILDKIT_SOURCE_POLICY=1`

Reply:
thanks! i changed this and forgot to update the doc